packages/mdc-chips/chip-set/foundation.ts

```diff
@@ -100,8 +106,50 @@ export class MDCChipSetFoundation extends MDCFoundation<MDCChipSetAdapter> {
    * Handles the event when a chip is removed.
    */
   handleChipRemoval(chipId: string) {
+    const index = this.adapter_.getIndexOfChipById(chipId);
```

Comment:
Yesterday we found the issue where removing a chip made 2 chips have tabIndex=0. This can be fixed by setting all tabIndex=-1, and then setting the next chip to tabIndex=0. I think this is the method that would contain that logic.

Reply:
Great find, thanks! I've fixed this.